src/main/java/mat/server/MappingSpreadsheetService.java

```diff
@@ -0,0 +1,107 @@
+package mat.server;
+
+import mat.client.shared.MatRuntimeException;
+import mat.server.spreadsheet.DataType;
+import mat.server.spreadsheet.MatAttribute;
+import mat.server.spreadsheet.QdmToQicoreMapping;
+import mat.server.spreadsheet.RequiredMeasureField;
+import mat.server.spreadsheet.ResourceDefinition;
+import org.apache.commons.lang3.StringUtils;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.cache.annotation.Cacheable;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Service;
+import org.springframework.web.client.RestClientResponseException;
+import org.springframework.web.client.RestTemplate;
+
+import javax.annotation.Resource;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Service
+public class MappingSpreadsheetService {
+    private static final String MAT_ATTRIBUTES = "/matAttributes";
+    private static final String QDM_TO_QICORE_MAPPING = "/qdmToQicoreMappings";
+    private static final String DATA_TYPES = "/dataTypes";
+    private static final String REQUIRED_MEASURE_FIELDS = "/requiredMeasureFields";
+    private static final String RESOURCE_DEFINITION = "/resourceDefinition";
+
+    @Value("${QDM_QICORE_MAPPING_SERVICES_URL:http://localhost:9090}")
+    private String fhirMatServicesUrl;
+
+    @Resource
+    MappingSpreadsheetService self;
```

Comment:
Codacy usually complains about package private fields

Reply:
it did and its fixed now.